localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -648,10 +652,10 @@ void EKFLocalizer::publishEstimateResult()
   twist_cov.header.stamp = current_time;
   twist_cov.header.frame_id = current_ekf_twist_.header.frame_id;
   twist_cov.twist.twist = current_ekf_twist_.twist;
-  twist_cov.twist.covariance[0] = P(IDX::VX, IDX::VX);
-  twist_cov.twist.covariance[5] = P(IDX::VX, IDX::WZ);
-  twist_cov.twist.covariance[30] = P(IDX::WZ, IDX::VX);
-  twist_cov.twist.covariance[35] = P(IDX::WZ, IDX::WZ);
+  twist_cov.twist.covariance[COV_IDX::X_X] = P(IDX::VX, IDX::VX);
```

Comment:
@scepter914 This value is the Twist, not the Pose.
So,  please modify `COV_IDX::X_X` to `TWIST_COV_IDX::VX_VX`).

Reply:
@YamatoAndo 
Thank you for review.
I modify covariance index name in https://github.com/autowarefoundation/autoware.universe/pull/1950.
So after merged this PR, I'll adjust the covariance index name.

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/1917/commits/6e728a50f0e5f5c98444345f70573f6048c6c80b